client/blocks/taxonomy-manager/list.jsx

```diff
@@ -68,15 +57,12 @@ export class TaxonomyManagerList extends Component {
 		}
 
 		// if item has a parent, and parent is in payload, height is already part of parent
-		if ( item.parent && ! _recurse && includes( this.termIds, item.parent ) ) {
+		if ( item.parent && ! _recurse && this.props.termIds.includes( item.parent ) ) {
 			return 0;
 		}
 
-		return reduce(
-			this.getTermChildren( item.ID ),
-			( memo, childItem ) => {
-				return memo + this.getItemHeight( childItem, true );
-			},
+		return this.getTermChildren( item.ID ).reduce(
+			( memo, childItem ) => memo + this.getItemHeight( childItem, true ),
```

Comment:
`memo` could be better called `sum` or `totalHeight`.

Reply:
👍🏻  fixed in https://github.com/Automattic/wp-calypso/pull/57819/commits/035e0f7ba387d7b0c1cdd89b438c6e0a3958a9d0